lib-ee/emqx_license/i18n/emqx_license_schema_i18n.conf

```diff
@@ -2,15 +2,13 @@ emqx_license_schema {
   license_root {
     desc {
       en: "Defines the EMQX Enterprise license. \n\n"
-          "A license is either a `key` or a `file`.\n"
-          "When `key` and `file` are both configured, `key` is used.\n"
+          "\n"
+          "The default license has 1000 connections limit, it is "
+          "issued on 2022-04-19 and valid for 5 years (1825 days).\n"
           "\n"
           "EMQX comes with a default trial license.  For production use, please \n"
           "visit https://www.emqx.com/apply-licenses/emqx to apply."
       zh: "EMQX企业许可证。\n"
-          "许可证是一个 `key` 或一个 `file`。\n"
-          "当 `key` 和 `file` 同时被配置时，优先使用 `key`。\n"
-          "\n"
           "EMQX 自带一个默认的试用许可证，若需要在生产环境部署，\n"
```

Comment:
```suggestion
          "EMQX 自带一个默认的试用许可证，"
          "默认试用许可允许最多接入 1000 个连接，签发时间是 2022年4月19日，有效期是 5 年（1825 天）。"
          "若需要在生产环境部署，\n"
```

Reply:
Oops, had forgotten to translate that :see_no_evil: 
Thanks!  :beers: